ish/ish.py

```diff
@@ -18,17 +18,22 @@
     u'لأ'.lower(),     # Arabic
 ]
 
+try:
+    basestring
+except NameError:
+    basestring = (str, bytes)
 
 class TrueIsh(object):
+    def _normalize_string(s):
+        if isinstance(other, bytes):
```

Comment:
This does work on Python 2.7 (maybe even 2.6) as well as on Python 3. That is because `bytes` has been added as alias for `str` on Python 2 in order to make supporting Python 2 and 3 with the same code easier.